sdk/netapp/arm-netapp/CHANGELOG.md

```diff
@@ -1,15 +1,60 @@
 # Release History
+    
+## 19.0.0 (2023-09-25)
+    
+**Features**
 
-## 18.1.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+  - Added operation NetAppResource.beginUpdateNetworkSiblingSet
+  - Added operation NetAppResource.beginUpdateNetworkSiblingSetAndWait
+  - Added operation NetAppResource.queryNetworkSiblingSet
+  - Added operation Volumes.beginPopulateAvailabilityZone
+  - Added operation Volumes.beginPopulateAvailabilityZoneAndWait
+  - Added Interface NetAppResourceQueryNetworkSiblingSetOptionalParams
+  - Added Interface NetAppResourceUpdateNetworkSiblingSetHeaders
+  - Added Interface NetAppResourceUpdateNetworkSiblingSetOptionalParams
+  - Added Interface NetworkSiblingSet
+  - Added Interface NicInfo
+  - Added Interface QueryNetworkSiblingSetRequest
+  - Added Interface UpdateNetworkSiblingSetRequest
+  - Added Interface VolumesPopulateAvailabilityZoneHeaders
+  - Added Interface VolumesPopulateAvailabilityZoneOptionalParams
+  - Added Type Alias CoolAccessRetrievalPolicy
+  - Added Type Alias NetAppResourceQueryNetworkSiblingSetResponse
+  - Added Type Alias NetAppResourceUpdateNetworkSiblingSetResponse
+  - Added Type Alias NetworkSiblingSetProvisioningState
+  - Added Type Alias VolumesPopulateAvailabilityZoneResponse
+  - Interface Volume has a new optional parameter coolAccessRetrievalPolicy
+  - Interface VolumeGroupVolumeProperties has a new optional parameter coolAccessRetrievalPolicy
+  - Interface VolumeGroupVolumeProperties has a new optional parameter zones
+  - Interface VolumePatch has a new optional parameter coolAccessRetrievalPolicy
+  - Interface VolumePatch has a new optional parameter smbAccessBasedEnumeration
+  - Interface VolumePatch has a new optional parameter smbNonBrowsable
+  - Added Enum KnownCoolAccessRetrievalPolicy
+  - Added Enum KnownNetworkSiblingSetProvisioningState
+  - Enum KnownApplicationType has a new value Oracle
+  - Enum KnownNetworkFeatures has a new value BasicStandard
+  - Enum KnownNetworkFeatures has a new value StandardBasic
 
-### Other Changes
+**Breaking Changes**
```

Comment:
swagger breaking pr https://github.com/Azure/azure-rest-api-specs/pull/25332

Reply:
we'll accept this breaking considering it is approved in swagger side.